2016-10-11 10:15:33,354 p=13 u=root |  PLAY [proxmox] *****************************************************************
2016-10-11 10:15:33,405 p=13 u=root |  TASK [Configure sipxcom] *******************************************************
2016-10-11 10:16:53,646 p=13 u=root |  changed: [10.3.0.60]
2016-10-11 10:16:53,655 p=13 u=root |  TASK [Start services] **********************************************************
2016-10-11 10:17:17,349 p=13 u=root |  changed: [10.3.0.60]
2016-10-11 10:17:17,358 p=13 u=root |  TASK [copy superadmin.sql function to remote machine] **************************
2016-10-11 10:30:54,786 p=13 u=root |  ok: [10.3.0.60]
2016-10-11 10:30:54,794 p=13 u=root |  TASK [Execute psql script] *****************************************************
2016-10-11 10:30:55,003 p=13 u=root |  changed: [10.3.0.60]
2016-10-11 10:30:55,016 p=13 u=root |  PLAY [localhost] ***************************************************************
2016-10-11 10:30:55,024 p=13 u=root |  TASK [setup] *******************************************************************
2016-10-11 10:30:55,946 p=13 u=root |  ok: [localhost]
2016-10-11 10:30:55,957 p=13 u=root |  TASK [Use REST API to populate servers] ****************************************
2016-10-11 10:30:56,197 p=13 u=root |  fatal: [localhost]: FAILED! => {"changed": true, "failed": true, "rc": 1, "stderr": "Traceback (most recent call last):\n  File \"/root/.ansible/tmp/ansible-tmp-1476181855.97-133502442748785/add_users_API.py\", line 11, in <module>\n    output_json = yaml.load(open('/files/json/users.json'))\nIOError: [Errno 2] No such file or directory: '/files/json/users.json'\n", "stdout": "", "stdout_lines": []}
2016-10-11 10:30:56,203 p=13 u=root |  NO MORE HOSTS LEFT *************************************************************
2016-10-11 10:30:56,204 p=13 u=root |  	to retry, use: --limit @/home/mcostache/PROIECTELE_MELE/Proxmox_Ansible/populate.retry

2016-10-11 10:30:56,205 p=13 u=root |  PLAY RECAP *********************************************************************
2016-10-11 10:30:56,205 p=13 u=root |  10.3.0.60                  : ok=4    changed=3    unreachable=0    failed=0   
2016-10-11 10:30:56,205 p=13 u=root |  localhost                  : ok=1    changed=0    unreachable=0    failed=1   
2016-10-11 11:17:28,894 p=12 u=root |  PLAY [proxmox] *****************************************************************
2016-10-11 11:17:28,917 p=12 u=root |  TASK [Configure sipxcom] *******************************************************
2016-10-11 11:21:23,485 p=12 u=root |  changed: [10.3.0.60]
2016-10-11 11:21:23,494 p=12 u=root |  TASK [Start services] **********************************************************
2016-10-11 11:21:48,621 p=12 u=root |  changed: [10.3.0.60]
2016-10-11 11:21:48,630 p=12 u=root |  TASK [copy superadmin.sql function to remote machine] **************************
2016-10-11 11:24:43,123 p=12 u=root |  fatal: [10.3.0.60]: FAILED! => {"changed": false, "cmd": "/usr/bin/rsync --delay-updates -F --compress --archive --rsh 'ssh  -S none -o StrictHostKeyChecking=no' --out-format='<<CHANGED>>%i %n%L' \"/home/mcostache/PROIECTELE_MELE/Proxmox_Ansible/files/json/superadmin.sql\" \"root@10.3.0.60:/var/log/superadmin.sql\"", "failed": true, "msg": "Connection to 10.3.0.60 closed by remote host.\r\nrsync: connection unexpectedly closed (0 bytes received so far) [sender]\nrsync error: unexplained error (code 255) at io.c(226) [sender=3.1.1]\n", "rc": 255}
2016-10-11 11:24:43,129 p=12 u=root |  NO MORE HOSTS LEFT *************************************************************
2016-10-11 11:24:43,130 p=12 u=root |  	to retry, use: --limit @/home/mcostache/PROIECTELE_MELE/Proxmox_Ansible/populate.retry

2016-10-11 11:24:43,130 p=12 u=root |  PLAY RECAP *********************************************************************
2016-10-11 11:24:43,131 p=12 u=root |  10.3.0.60                  : ok=2    changed=2    unreachable=0    failed=1   
2016-10-11 11:43:25,144 p=12 u=root |  PLAY [proxmox] *****************************************************************
2016-10-11 11:43:25,165 p=12 u=root |  TASK [Configure sipxcom] *******************************************************
2016-10-11 11:47:09,866 p=12 u=root |  changed: [10.3.0.60]
2016-10-11 11:47:09,875 p=12 u=root |  TASK [Start services] **********************************************************
2016-10-11 11:47:34,689 p=12 u=root |  changed: [10.3.0.60]
2016-10-11 11:47:34,698 p=12 u=root |  TASK [copy superadmin.sql function to remote machine] **************************
2016-10-11 11:48:40,916 p=12 u=root |  fatal: [10.3.0.60]: FAILED! => {"changed": false, "cmd": "/usr/bin/rsync --delay-updates -F --compress --archive --rsh 'ssh  -S none -o StrictHostKeyChecking=no' --out-format='<<CHANGED>>%i %n%L' \"/home/mcostache/PROIECTELE_MELE/Proxmox_Ansible/files/json/superadmin.sql\" \"root@10.3.0.60:/var/log/superadmin.sql\"", "failed": true, "msg": "Permission denied, please try again.\r\nbash: rsync: command not found\nrsync: connection unexpectedly closed (0 bytes received so far) [sender]\nrsync error: remote command not found (code 127) at io.c(226) [sender=3.1.1]\n", "rc": 127}
2016-10-11 11:48:40,922 p=12 u=root |  NO MORE HOSTS LEFT *************************************************************
2016-10-11 11:48:40,925 p=12 u=root |  	to retry, use: --limit @/home/mcostache/PROIECTELE_MELE/Proxmox_Ansible/populate.retry

2016-10-11 11:48:40,925 p=12 u=root |  PLAY RECAP *********************************************************************
2016-10-11 11:48:40,925 p=12 u=root |  10.3.0.60                  : ok=2    changed=2    unreachable=0    failed=1   
2016-10-11 11:51:03,877 p=13 u=root |  ERROR! 'force' is not a valid attribute for a Task

The error appears to have been in '/home/mcostache/PROIECTELE_MELE/Proxmox_Ansible/populate.yml': line 20, column 5, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


  - name: copy superadmin.sql function to remote machine
    ^ here

2016-10-11 11:51:25,624 p=13 u=root |  ERROR! Syntax Error while loading YAML.


The error appears to have been in '/home/mcostache/PROIECTELE_MELE/Proxmox_Ansible/populate.yml': line 22, column 15, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

    copy: src="files/json/superadmin.sql"  dest="/var/log/superadmin.sql"
      validate: no
              ^ here

2016-10-11 11:51:40,846 p=13 u=root |  ERROR! 'validate' is not a valid attribute for a Task

The error appears to have been in '/home/mcostache/PROIECTELE_MELE/Proxmox_Ansible/populate.yml': line 20, column 5, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


  - name: copy superadmin.sql function to remote machine
    ^ here

2016-10-11 11:52:31,681 p=12 u=root |  PLAY [proxmox] *****************************************************************
2016-10-11 11:52:31,705 p=12 u=root |  TASK [Configure sipxcom] *******************************************************
2016-10-11 11:56:20,931 p=12 u=root |  changed: [10.3.0.60]
2016-10-11 11:56:20,939 p=12 u=root |  TASK [Start services] **********************************************************
2016-10-11 11:56:46,398 p=12 u=root |  changed: [10.3.0.60]
2016-10-11 11:56:46,406 p=12 u=root |  TASK [copy superadmin.sql function to remote machine] **************************
2016-10-11 12:08:20,839 p=12 u=root |  fatal: [10.3.0.60]: FAILED! => {"changed": false, "checksum": "66342c81d030999e4b9ad514a0fda0f62db97bc9", "failed": true, "msg": "validate must contain %s: no"}
2016-10-11 12:08:20,845 p=12 u=root |  NO MORE HOSTS LEFT *************************************************************
2016-10-11 12:08:20,846 p=12 u=root |  	to retry, use: --limit @/home/mcostache/PROIECTELE_MELE/Proxmox_Ansible/populate.retry

2016-10-11 12:08:20,846 p=12 u=root |  PLAY RECAP *********************************************************************
2016-10-11 12:08:20,846 p=12 u=root |  10.3.0.60                  : ok=2    changed=2    unreachable=0    failed=1   
2016-10-11 12:12:01,067 p=13 u=root |  PLAY [proxmox] *****************************************************************
2016-10-11 12:12:01,087 p=13 u=root |  TASK [Configure sipxcom] *******************************************************
2016-10-11 12:15:52,889 p=13 u=root |  changed: [10.3.0.60]
2016-10-11 12:15:52,897 p=13 u=root |  TASK [Start services] **********************************************************
2016-10-11 12:16:17,492 p=13 u=root |  changed: [10.3.0.60]
2016-10-11 12:16:17,501 p=13 u=root |  TASK [Run psql.sh Script] ******************************************************
2016-10-11 12:16:17,794 p=13 u=root |  fatal: [10.3.0.60]: FAILED! => {"changed": true, "failed": true, "rc": 1, "stderr": "", "stdout": "files/configuration/superadmin.sql: No such file or directory\r\n", "stdout_lines": ["files/configuration/superadmin.sql: No such file or directory"]}
2016-10-11 12:16:17,800 p=13 u=root |  NO MORE HOSTS LEFT *************************************************************
2016-10-11 12:16:17,803 p=13 u=root |  	to retry, use: --limit @/home/mcostache/PROIECTELE_MELE/Proxmox_Ansible/populate.retry

2016-10-11 12:16:17,804 p=13 u=root |  PLAY RECAP *********************************************************************
2016-10-11 12:16:17,804 p=13 u=root |  10.3.0.60                  : ok=2    changed=2    unreachable=0    failed=1   
2016-10-11 12:20:07,904 p=13 u=root |  PLAY [proxmox] *****************************************************************
2016-10-11 12:20:07,925 p=13 u=root |  TASK [Configure sipxcom] *******************************************************
2016-10-11 12:24:02,145 p=13 u=root |  changed: [10.3.0.60]
2016-10-11 12:24:02,154 p=13 u=root |  TASK [Start services] **********************************************************
2016-10-11 12:24:30,304 p=13 u=root |  changed: [10.3.0.60]
2016-10-11 12:24:30,311 p=13 u=root |  TASK [Run psql.sh Script] ******************************************************
2016-10-11 12:24:30,596 p=13 u=root |  changed: [10.3.0.60]
2016-10-11 12:24:30,609 p=13 u=root |  PLAY [localhost] ***************************************************************
2016-10-11 12:24:30,617 p=13 u=root |  TASK [setup] *******************************************************************
2016-10-11 12:24:31,509 p=13 u=root |  ok: [localhost]
2016-10-11 12:24:31,517 p=13 u=root |  TASK [Use REST API to populate servers] ****************************************
2016-10-11 12:24:32,277 p=13 u=root |  changed: [localhost]
2016-10-11 12:24:32,284 p=13 u=root |  PLAY RECAP *********************************************************************
2016-10-11 12:24:32,285 p=13 u=root |  10.3.0.60                  : ok=3    changed=3    unreachable=0    failed=0   
2016-10-11 12:24:32,285 p=13 u=root |  localhost                  : ok=2    changed=1    unreachable=0    failed=0   
2016-10-11 12:30:30,190 p=13 u=root |  PLAY [proxmox] *****************************************************************
2016-10-11 12:30:30,213 p=13 u=root |  TASK [Configure sipxcom] *******************************************************
2016-10-11 12:34:17,229 p=13 u=root |  changed: [10.3.0.60]
2016-10-11 12:34:17,239 p=13 u=root |  TASK [Start services] **********************************************************
2016-10-11 12:34:41,773 p=13 u=root |  changed: [10.3.0.60]
2016-10-11 12:34:41,780 p=13 u=root |  TASK [Run psql.sh Script] ******************************************************
2016-10-11 12:34:42,025 p=13 u=root |  fatal: [10.3.0.60]: FAILED! => {"changed": true, "failed": true, "rc": 1, "stderr": "", "stdout": "ERROR:  syntax error at or near \"23122\"\r\nLINE 3: CREATE FUNCTION create_superadmin() RETURNS integer AS 23122\r\n                                                               ^\r\n", "stdout_lines": ["ERROR:  syntax error at or near \"23122\"", "LINE 3: CREATE FUNCTION create_superadmin() RETURNS integer AS 23122", "                                                               ^"]}
2016-10-11 12:34:42,031 p=13 u=root |  NO MORE HOSTS LEFT *************************************************************
2016-10-11 12:34:42,032 p=13 u=root |  	to retry, use: --limit @/home/mcostache/PROIECTELE_MELE/Proxmox_Ansible/populate.retry

2016-10-11 12:34:42,032 p=13 u=root |  PLAY RECAP *********************************************************************
2016-10-11 12:34:42,032 p=13 u=root |  10.3.0.60                  : ok=2    changed=2    unreachable=0    failed=1   
2016-10-11 12:38:16,890 p=12 u=root |  PLAY [proxmox] *****************************************************************
2016-10-11 12:38:16,910 p=12 u=root |  TASK [Configure sipxcom] *******************************************************
2016-10-11 12:42:02,436 p=12 u=root |  changed: [10.3.0.60]
2016-10-11 12:42:02,445 p=12 u=root |  TASK [Start services] **********************************************************
2016-10-11 12:42:29,821 p=12 u=root |  changed: [10.3.0.60]
2016-10-11 12:42:29,831 p=12 u=root |  TASK [copy superadmin.sql function to remote machine] **************************
2016-10-11 12:42:29,862 p=12 u=root |  fatal: [10.3.0.60]: FAILED! => {"failed": true, "msg": "async mode is not supported with the copy module"}
2016-10-11 12:42:29,868 p=12 u=root |  NO MORE HOSTS LEFT *************************************************************
2016-10-11 12:42:29,870 p=12 u=root |  	to retry, use: --limit @/home/mcostache/PROIECTELE_MELE/Proxmox_Ansible/populate.retry

2016-10-11 12:42:29,870 p=12 u=root |  PLAY RECAP *********************************************************************
2016-10-11 12:42:29,870 p=12 u=root |  10.3.0.60                  : ok=2    changed=2    unreachable=0    failed=1   
2016-10-11 12:45:04,895 p=13 u=root |  PLAY [proxmox] *****************************************************************
2016-10-11 12:45:04,917 p=13 u=root |  TASK [Configure sipxcom] *******************************************************
2016-10-11 12:48:59,485 p=13 u=root |  changed: [10.3.0.60]
2016-10-11 12:48:59,493 p=13 u=root |  TASK [Start services] **********************************************************
2016-10-11 12:49:24,821 p=13 u=root |  changed: [10.3.0.60]
2016-10-11 12:49:24,830 p=13 u=root |  TASK [copy superadmin.sql function to remote machine] **************************
2016-10-11 13:00:58,355 p=13 u=root |  changed: [10.3.0.60]
2016-10-11 13:00:58,363 p=13 u=root |  TASK [Execute psql script] *****************************************************
2016-10-11 13:00:58,688 p=13 u=root |  changed: [10.3.0.60]
2016-10-11 13:00:58,700 p=13 u=root |  PLAY [localhost] ***************************************************************
2016-10-11 13:00:58,704 p=13 u=root |  TASK [setup] *******************************************************************
2016-10-11 13:00:59,591 p=13 u=root |  ok: [localhost]
2016-10-11 13:00:59,600 p=13 u=root |  TASK [Use REST API to populate servers] ****************************************
2016-10-11 13:01:18,934 p=13 u=root |  changed: [localhost]
2016-10-11 13:01:18,940 p=13 u=root |  PLAY RECAP *********************************************************************
2016-10-11 13:01:18,941 p=13 u=root |  10.3.0.60                  : ok=4    changed=4    unreachable=0    failed=0   
2016-10-11 13:01:18,941 p=13 u=root |  localhost                  : ok=2    changed=1    unreachable=0    failed=0   
2016-10-11 13:11:44,163 p=13 u=root |  PLAY [proxmox] *****************************************************************
2016-10-11 13:11:44,190 p=13 u=root |  TASK [setup] *******************************************************************
2016-10-11 13:11:46,759 p=13 u=root |  ok: [10.3.0.60]
2016-10-11 13:11:46,768 p=13 u=root |  TASK [copy superadmin.sql function to remote machine] **************************
2016-10-11 13:13:55,983 p=13 u=root |   [ERROR]: User interrupted execution

